projects/igniteui-angular/src/lib/grids/public_api.ts

```diff
@@ -13,7 +13,7 @@ export * from './grid-base.directive';
 export * from './grid.common';
 export * from './grid-public-row';
 export * from './grid-public-cell';
-export { CellType, RowType, IGX_GRID_BASE } from './common/grid.interface';
+export { CellType, RowType, IGX_GRID_BASE, Validity, IRecordValidationState, IFieldValidationState } from './common/grid.interface';
 export * from './summaries/grid-summary';
 export * from './grid-common.module';
 export * from './grid.rowEdit.directive';
```

Comment:
Is the Validation service exposed somewhere in the public API? Because I don't see it and it is part of the Grid's public API

Reply:
It's exposed now,